langchain/src/agents/chat_convo/index.ts

```diff
@@ -49,10 +50,14 @@ export class ChatConversationalAgentOutputParser extends BaseOutputParser {
 }
 
 export type CreatePromptArgs = {
-  /** String to put after the list of tools. */
+  /** DEPRECATED: String to put after the list of tools. */
   systemMessage?: string;
-  /** String to put before the list of tools. */
+  /** DEPRECATED: String to put before the list of tools. */
   humanMessage?: string;
```

Comment:
I opted to deprecate these properties to be in line with the other agents - all prompt args should probably all inherit from the same type anyway, no?

Reply:
So these have different arg names because they are designed to work with chat models, instead of string LLMs as the other agents. Hence the different arg names. Thoughts?

Reply:
Ah ok understood, thanks for the context. What would you think of subclassing it into a `ChatAgent`?

Happy to revert and leave it the way it was if you prefer that.

Reply:
Added a bit more discussion here: https://github.com/hwchase17/langchainjs/pull/374#discussion_r1151175672